### PR TITLE
Added example to app activity differences

### PR DESCRIPTION
**Pull request by MichaelKowal**

### CODE REVIEW

**Comment by MichaelKowal** [comment]
PR for https://dev.azure.com/geocortex/Geocortex/_workitems/edit/40252

**Comment by Noah-Silvera** [comment]
@MichaelKowal the prettier check failed - could you try running `npm run prettier:write` manually then run `npm lint` to make sure there's no issues before you push?

**Comment by Noah-Silvera** [comment]
@MichaelKowal I realize now that the user story implied that the text was supposed to be in italic.... I just was highlighting the new text with italic for clarity in the ticket, but it should actually just be normal font and styling like the other text in the note. Thanks!

File: docs/workflow/usecases-web-access-app-properties.mdx [hunk 1]
```diff
@@ -12,7 +12,7 @@ Learn how to build a [custom activity](usecases-web-implement-custom-activity.md
 Some activities may need access to resources like the map, the host application or host widget. The `AppActivity` base class can be used to extend an existing custom activity to access these resources.
 
 :::note
-Depending on the type of host application different resources will be available.
+Depending on the type of host application different resources will be available. _For example, Web AppBuilder for ArcGIS is the only platform which has access to the `widget` property._
```
Comment (Noah-Silvera) [inline docs/workflow/usecases-web-access-app-properties.mdx:15]:
```suggestion
Depending on the type of host application different resources will be available. For example, Web AppBuilder for ArcGIS is the only platform which has access to the `widget` property.
```

**Pull request merged**